src/styles/abstracts/_variables.scss

```diff
@@ -112,6 +113,12 @@ $modal-title-margin-left: $modal-close-size + ($modal-close-margin * 2);
 $card-edge-size: 1px;
 $card-offset: 2rem;
 
+/**
+ * Card
+ */
+
+$button-horizontal-whitespace: $spacer-md;
```

Comment:
Waarom gebruik je niet gewoon `$spacer-md`?

Reply:
Maintainability, `$spacer-md` gebruiken in `src/modules/shared/components/PaginationBar/PaginationBar.module.scss` zou niet duidelijk maken waar de waarde eigenlijk voor dient. Het "magic number" probleem.